Source/JavaScriptCore/dfg/DFGJITCode.h

```diff
@@ -77,6 +85,7 @@ class JITData final : public TrailingArray<JITData, void*> {
     }
 
     ExitVector m_exits;
+    uint8_t m_invalidation { 0 };
```

Comment:
can we call this "m_isInvalidated" and functions above.

Reply:
Fixed.